internal/configuration/platform/export.go

```diff
@@ -168,12 +177,14 @@ func initPingOneEnvironmentIDOption() {
 		DefaultValue:    &defaultValue,
 		EnvVar:          envVar,
 		Flag: &pflag.Flag{
+			Annotations: map[string][]string{
+				"KoanfKey": {"pingone.environmentID"},
```

Comment:
`pingone.environmentID` and `export.pingOne.environmentID` do not match.

For all options using auto-complete annotation, use variable to change koanf key value in one place only

Reply:
These aren't used and were an artifact of a previous idea of implementation - will remove all of these as they're not used.